lib/include/subzero/autonomous/AutoFactory.h

```diff
@@ -34,31 +34,9 @@ template <typename T> class AutoFactory {
 private:
   const std::map<T, std::string> &m_autos;
 
-  bool AutoFileExists(const std::string fileName) {
-    const std::string filePath = frc::filesystem::GetDeployDirectory() +
-                                 "/pathplanner/autos/" + fileName + ".auto";
+  bool AutoFileExists(const std::string fileName);
 
-    // Use wpi::MemoryBuffer::GetFile and handle the wpi::expected result
-    auto fileBufferResult = wpi::MemoryBuffer::GetFile(filePath);
-
-    if (!fileBufferResult) {
-      // fileBufferResult is an error; the file does not exist
-      return false;
-    }
-
-    // fileBufferResult contains a valid std::unique_ptr<wpi::MemoryBuffer>
-    return true;
-  }
-
-  frc2::CommandPtr PathPlannerPathFromName(const std::string autoName) {
-    if (!AutoFileExists(autoName)) {
-      ConsoleWriter.logError("Auto Factory",
-                             "AUTO '%s' DOES NOT EXIST HELP US EVAN",
-                             autoName.c_str());
-      return EmptyCommand().ToPtr();
-    }
-    return pathplanner::PathPlannerAuto(autoName).ToPtr();
-  }
+  frc2::CommandPtr PathPlannerPathFromName(const std::string autoName);
```

Comment:
Make sure if you're removing the header only implementation that you have a cpp somewhere so you don't get a linking error